Abstract Syntax and Semantics Document/index.html

```diff
@@ -398,9 +398,89 @@
       <p>See <a href="https://github.com/streamreasoning/RSP-QL/issues/11">Issue 11</a>.</p>
     </blockquote>
     </section>
-    <section>
+    <section id="merge-union">
     
-    <h2>Stream Merger and Union Operations</h2>
+    <h2>Stream Merge and Union Operations</h2>
+      <p>In order to combine two RDF streams into one, without loss or gratuitous introduction of entailments, we define two relations, <em>RDF stream merge</em> and <em>RDF stream union</em>.
+        These relations are inspired by the concepts of similar name for RDF graphs defined in [[RDF11-Concepts]].
+      </p>
+      <p>
+        <em>RDF stream union</em> is a trinary relation on RDF streams. Let <code>S1</code>, <code>S2</code>, and <code>S3</code> be RDF streams.
+        We say <code>S3</code> is an <em>RDF stream union</em> of <code>S1</code> and <code>S2</code> if and only if the set of elements of <code>S3</code> is equal to the union of the sets of elements of <code>S1</code> with <code>S2</code>.</p>
+      <p class="note">
+        Union is the appropriate relation to consider when combining streams that are allowed to share blank nodes. Informally, they would be on the same RDF surface.
```

Comment:
The intuitive notion of RDF surface is introduce by a link in the isomorphism section. It doesn't show up in the diff because that came from a different issue, but the link is file:///home/tara/Repositories/Git/RSP-QL/Abstract%20Syntax%20and%20Semantics%20Document/index.html#bib-BLOGIC . The idea is that triples that are on the same RDF surface can share blank nodes, otherwise they cannot. The term "RDF surface" is only used in informative text - it does not play a role in the normative definitions.